libyara/modules/dex/dex.c

```diff
@@ -37,6 +37,165 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 // DEX File layout information:
 // https://source.android.com/devices/tech/dalvik/dex-format
 
+define_function(has_method_string)
+{
+  int i;
+  SIZED_STRING* parsed_name;
+  SIZED_STRING* method_name = sized_string_argument(1);
+  YR_OBJECT* module = module();
+  int64_t number_of_methods = get_integer(module, "number_of_methods");
+
+  if (number_of_methods == UNDEFINED)
+    return_integer(UNDEFINED);
+
+  for (i = 0; i < number_of_methods; i++)
```

Comment:
We can start using C99 syntax and define the `i` identifier here `for (int i = 0 ..`